cycode/cli/commands/scan/code_scanner.py

```diff
@@ -312,17 +312,36 @@ def scan_documents(
         )
         return
 
-    scan_batch_thread_func = _get_scan_documents_thread_func(context, is_git_diff, is_commit_range, scan_parameters)
+    scan_batch_thread_func, aggregation_id = _get_scan_documents_thread_func(
+        context, is_git_diff, is_commit_range, scan_parameters
+    )
     errors, local_scan_results = run_parallel_batched_scan(
         scan_batch_thread_func, documents_to_scan, progress_bar=progress_bar
     )
+    aggregation_report_url = _try_get_aggregation_report_url_if_needed(
+        scan_parameters, context.obj['client'], context.obj['scan_type'], aggregation_id
```

Comment:
we already have `aggregation_id` in the `scan_parameters`. i think we should not have duplicated args